database/bigMenuData.js

```diff
@@ -2,67 +2,68 @@ const faker = require('faker');
 const fs = require('file-system');
 const path = require('path');
 const csvWriter = require('csv-write-stream');
-const headers =  {
-  headers: [
-    'id',
-    'mealType',
-    'Item',
-    'Des',
-    'Price'
-  ]
+const headers = {
+  headers: ['Breakfast', 'Lunch', 'Dinner', 'Brunch', 'HappyHour']
 }
 const writer = csvWriter(headers);
 
+  var generateMenuObject_100000_helper = function() {
+    result = {};
+    var result = {'Breakfast':[], 'Lunch':[], 'Dinner':[], 'Brunch': [], 'HappyHour':[]};
```

Comment:
I'm thinking these nested arrays are probably a significant factor in lengthening your script run-times. I'm not sure of a specific redesign that might help, but I think it's worth looking into.